app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,279 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Api
+  extend ActiveSupport::Concern
+
+  included do
+    def dummy_client
+      @dummy_client ||= ExTwitter::Client.new
+    end
+
+    def one_sided_following
+      dummy_client.one_sided_following(self)
+    end
+
+    def one_sided_followers
+      dummy_client.one_sided_followers(self)
+    end
+
+    def mutual_friends
+      dummy_client.mutual_friends(self)
+    end
+
+    def common_friends(other)
+      return [] if other.blank?
+      dummy_client.common_friends(self, other)
+    end
+
+    def common_followers(other)
+      return [] if other.blank?
+      dummy_client.common_followers(self, other)
+    end
+
+    def removing
+      return [] unless TwitterUser.has_more_than_two_records?(uid, user_id)
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        dummy_client.removing(old_one, new_one)
+      end.flatten
+    end
+
+    def removed
+      return [] unless TwitterUser.has_more_than_two_records?(uid, user_id)
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        dummy_client.removed(old_one, new_one)
+      end.flatten
+    end
+
+    def replying(options = {})
+      if statuses.any?
+        client.replying(statuses.to_a, options)
+      else
+        client.replying(uid.to_i, options)
+      end.map { |u| u.uid = u.id; u }
+    end
+
+    def replied(options = {})
+      result =
+        if ego_surfing?
+          if mentions.any?
+            mentions.map { |m| m.user }.map { |u| u.uid = u.id; u }
```

Comment:
__[rubocop]__
* Pass `&:user` as an argument to `map` instead of a block.
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)